src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -23,18 +23,18 @@ namespace node {
 
 #if defined(OPENVINO_ARCH_X86_64)
 namespace {
-struct jit_has_subnormals_base : public jit_generator {
-    DECLARE_CPU_JIT_AUX_FUNCTIONS(jit_has_subnormals_base)
+struct jit_subnormals_bf16saturation_check_base : public jit_generator {
+    DECLARE_CPU_JIT_AUX_FUNCTIONS(jit_subnormals_bf16saturation_check_base)
```

Comment:
the name is getting longer, can we use more generic name like `jit_has_special_value_base` ?

Reply:
done